### ISSUE TITLE
Nvidia support - Update training profiles to use FSDP as default

### ISSUE THREAD

**Issue by ktam3** [instructlab/instructlab#2521]
Blocked on: 
- [ ] https://github.com/instructlab/training/issues/292

**Comment by nathan-weinberg** [comment]
I _think_ @jaideepr97 has already done this with https://github.com/instructlab/instructlab/pull/2323? The issue is a little vague tbh so not sure. cc @cdoern
